contracts/tokenomics/reserve/src/contract.rs

```diff
@@ -20,12 +20,16 @@ use cosmwasm_std::{
     coins, to_binary, Addr, BankMsg, Binary, Coin, CosmosMsg, Decimal, Deps, DepsMut, Env,
     MessageInfo, Response, StdResult, Uint128, WasmMsg,
 };
+use cw2::set_contract_version;
 use cw20::{BalanceResponse, Cw20ExecuteMsg, Cw20QueryMsg};
 use exec_control::pause::{
     can_pause, can_unpause, validate_duration, PauseError, PauseInfoResponse,
 };
 use neutron_sdk::bindings::query::NeutronQuery;
 
+pub(crate) const CONTRACT_NAME: &str = "crates.io:reserve";
```

Comment:
In `Cargo.toml` we have name of the contract equals to `neutron-reserve`. Let's keep things consistent and have the same name here.